assets/stylesheets/tables.scss

```diff
@@ -63,7 +63,11 @@
         text-align: center;
     }
 }
-
+table.dataTable tr {
+    td span {
+        white-space: pre;
```

Comment:
This rule seems to generic. I don't think we want to match *every* span (which is one of the most generic elements) in *every* data-table.

Reply:
Yeah, I was kind of expecting that it would be to generic.

Reply:
I added a settings-table class now